src/Lykke.Numerics.Money/UMoney.Arithmetic.cs

```diff
@@ -207,14 +206,17 @@ public partial struct UMoney
         /// <returns>
         ///    The result of subtracting right from left.
         /// </returns>
+        /// <exception cref="OverflowException">
+        ///    Result is lower than zero.
+        /// </exception>
         [Pure]
         public static UMoney Subtract(
             UMoney left,
             UMoney right)
         {
             if (right > left)
             {
-                throw new InvalidOperationException("Right should be lower or equal to the left.");
+                throw new OverflowException();
```

Comment:
Would be good to have some meaningful message in the exception

Reply:
OverflowException does not allow us to specify a message.